### PR TITLE
Index Management underestimate number of documents in the index to rollover

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1861009

https://coreos.slack.com/archives/GGUR75P60/p1594916413082600

### CODE REVIEW

**Comment by mburke5678** [comment]
@lukas-vlcek @QiaolingTang Please take a look. PR to change the reference to the number of index documents that triggers Elasticsearch rollover

**Review by anpingli — COMMENTED** [review]
LGTM

File: modules/cluster-logging-elasticsearch-retention.adoc [hunk 1]
```diff
@@ -16,8 +16,8 @@ Elasticsearch rolls over an index, moving the current index and creating a new
 index, when an index matches any of the following conditions:
 
 * The index is older than than the `rollover.maxAge` value in the Elasticsearch CR.
-* The index size is greater than 40gb x number of primary shards.
-* The index doc count is greater than 40960 x number of primary shards.
+* The index size is greater than 40gb x the number of primary shards.
+* The index doc count is greater than 40960k x the number of primary shards.
```
Comment (adellape) [inline modules/cluster-logging-elasticsearch-retention.adoc:20]: Per IBM SG seems the style should be:

40 GB × the number [...]
40960 KB × the number [...]

Uppercase abbrev with a space and with (×) being an actual multiplication sign.

**Pull request merged**